driver/controller.go

```diff
@@ -806,11 +806,30 @@ func (d *Driver) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsReques
 	return listResp, nil
 }
 
+// ControllerExpandVolume is called from the resizer to increase the volume size.
 func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	d.log.WithField("method", "controller_expand_volume").
-		Info("controller expand volume called")
```

Comment:
We should retain this log and add more information to it, similar to what we do in the [controller Create RPC](https://github.com/digitalocean/csi-digitalocean/blob/master/driver/controller.go#L114). These messages are super helpful when debugging CSI issues.